distribution/docs/src/main/resources/_contents/_transformers/registry-transformer-contents.adoc

```diff
@@ -0,0 +1,20 @@
+==== Registry Transformer
+
+The Registry Transformer is responsible for translating a {document} into a Registry Metacard.
+
+===== Installing Registry Transformer
+
+The Registry Transformer is installed by default with a standard installation in the ${app-name} application.
```

Comment:
This one is correct if we are talking about the Registry App being installed, not the one below.